docs/source/deployment/airflow_astronomer.md

```diff
@@ -2,161 +2,153 @@
-example_test_x:
-  type: pickle.PickleDataset
-  filepath: data/05_model_input/example_test_x.pkl
-example_test_y:
-  type: pickle.PickleDataset
-  filepath: data/05_model_input/example_test_y.pkl
-example_model:
-  type: pickle.PickleDataset
-  filepath: data/06_models/example_model.pkl
-example_predictions:
-  type: pickle.PickleDataset
-  filepath: data/07_model_output/example_predictions.pkl
+  filepath: data/02_intermediate/y_test.csv
 ```
 
-This ensures that all datasets are persisted so all Airflow tasks can read them without the need to share memory. In the example here we assume that all Airflow tasks share one disk, but for distributed environment you would need to use non-local filepaths.
+In the example here we assume that all Airflow tasks share one disk, but for distributed environments you would need to use non-local file paths.
 
-#### Step 2. Package the Kedro pipeline as an Astronomer-compliant Docker image
+4. Open `conf/logging.yml` and modify the `root: handlers` section to `[console]` at the end of the file. By default, Kedro uses the [Rich library](https://rich.readthedocs.io/en/stable/index.html) to enhance log output with sophisticated formatting. However, some deployment systems, including Airflow, don't work well with Rich. Therefore, we're adjusting the logging to a simpler console version. For more information on logging in Kedro, you can refer to the [Kedro docs](https://docs.kedro.org/en/stable/logging/index.html).
 
-* **Step 2.1**: Package the Kedro pipeline as a Python package so you can install it into the container later on:
+```shell
+root:
+  handlers: [console]
+```
+
+5. Package the Kedro pipeline as a Python package so you can install it into the Airflow container later on:
 
 ```shell
 kedro package
 ```
 
 This step should produce a wheel file called `new_kedro_project-0.1-py3-none-any.whl` located at `dist/`.
 
-* **Step 2.2**: Add the `src/` directory to `.dockerignore`, as it's not necessary to bundle the entire code base with the container once we have the packaged wheel file.
+6. Convert the Kedro pipeline into an Airflow DAG with `kedro airflow`
 
 ```shell
-echo "src/" >> .dockerignore
+kedro airflow create --target-dir=dags/ --env=airflow
 ```
 
-* **Step 2.3**: Modify the `Dockerfile` to have the following content:
+This step should produce a .py file called `new_kedro_project_dag.py` located at `dags/`.
+
+### Deployment process with Astro Airflow
+
+In this section, we'll start by setting up a new blank Airflow project using Astro. We'll then copy the files prepared in the previous section from our Kedro project. Next, we'll customize the Dockerfile to enhance logging capabilities and manage the installation of our Kedro package. Finally, we will run and explore the Airflow cluster.
+
+1. [Initialise an Airflow project with Astro](https://docs.astronomer.io/astro/cli/develop-project) in a new folder outside of your Kedro project. Let's call it `kedro-airflow-spaceflights`
```

Comment:
I followed through the tutorial and got stuck at this step because I didn't have the `astro` CLI installed. It would be good to mention it needs to be installed.

Reply:
@merelcht , we have a section Prerequisites, where that is mentioned, do you think that it's not enough? Also when you will click the link and try to install Astro CLI it will be mentioned there that you need Docker for that:

Prerequisites
To follow this tutorial, ensure you have the following:

The Astro CLI installed

Reply:
Ah yes I read over the Astro CLI part. I think it won't hurt to mention again at this point in the tutorial you need the Astro CLI and Docker. I would personally also mention Docker explicitly in the prereqs.

Reply:
Agreed, I've included an additional note about Docker and the Astro CLI at the start of section 2 and also mentioned Docker in the prerequisites.